demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -660,7 +660,8 @@ static int establishMqttSession( MQTTContext_t * pMqttContext,
     MQTTStatus_t mqttStatus;
     MQTTConnectInfo_t connectInfo;
     MQTTFixedBuffer_t networkBuffer;
-    MQTTApplicationCallbacks_t callbacks;
+    MQTTEventCallback_t userCallback;
```

Comment:
Are these two local variables needed anymore? Can we not directly have:

```c
MQTT_Init( pMqttContext, &transport, Clock_GetTimeMs, eventCallback, &networkBuffer );
```

Reply:
I added the local variables only for keeping the verbose documentation
```
  /* Application callbacks for receiving incoming publishes and incoming acks
     * from MQTT library. */
    callbacks.appCallback = eventCallback;

    /* Application callback for getting the time for MQTT library. This time
     * function will be used to calculate intervals in MQTT library.*/
    callbacks.getTime = Clock_GetTimeMs;
```